scripts/resolve_i64/main.go

```diff
@@ -25,22 +25,22 @@ import (
 	"os"
 
 	//"github.com/uber/zanzibar/codegen"
+	"bytes"
+	"github.com/pkg/errors"
+	"github.com/uber/zanzibar/codegen"
+	"github.com/uber/zanzibar/runtime"
 	"go.uber.org/thriftrw/compile"
 	"html/template"
-	"bytes"
 	"io/ioutil"
-	"github.com/uber/zanzibar/runtime"
-	"runtime"
 	"path/filepath"
-	"strings"
-	"github.com/uber/zanzibar/codegen"
-	"github.com/pkg/errors"
+	"runtime"
```

Comment:
since you are at it, let's group all stdlibs to the first import group

Reply:
will do.